www/artists/delete.php

```diff
@@ -0,0 +1,82 @@
+<?
+use function Safe\session_unset;
+
+$artist = null;
+
+try{
+	if(Session::$User === null){
+		throw new Exceptions\LoginRequiredException();
+	}
+
+	if(!Session::$User->Benefits->CanReviewOwnArtwork){
+		throw new Exceptions\InvalidPermissionsException();
+	}
+
+	session_start();
+
+	$exception = HttpInput::SessionObject('exception', Exceptions\AppException::class);
+
+	$artist = Artist::GetByUrlName(HttpInput::Str(GET, 'artist-url-name') ?? '');
+	$artworkCount = count(Artwork::GetAllByArtist($artist->UrlName, Enums\ArtworkFilterType::Admin, null /* submitterUserId */));
```

Comment:
I don't think we need to calculate `$artworkCount`, it's a DB call and the number is not especially relevant to anything

Reply:
Done